.github/workflows/release.yml

```
@@ -21,6 +21,12 @@ jobs:
         uses: percona-platform/setup-go@v2
         with:
           go-version: 1.16
+      - 
+        name: Login to Docker Hub
+        uses: docker/login-action@v1
```

```suggestion
        uses: percona-platform/login-action@v1
```

we fork it for the security reasons.